public/stylesheet.css

```
@@ -397,13 +396,23 @@ body.no-js .control-field__title {
 }
 
 .control__button {
+  border-bottom: 0.1rem solid var(--color-button-border);
+  border-top: 0.1rem solid var(--color-button-border);
+  border-right: 0.1rem solid var(--color-button-border);
   cursor: default; /* Button should not work when JS is disabled */
   fill: var(--color-button-hover);
-  margin: 0 0.3rem;
   width: 2.4rem;
 }
 .control__button:first-of-type {
   margin-left: 0;
+  border-left: 0.1rem solid var(--color-button-border);
+  border-top-left-radius: var(--border-radius-small);
+  border-bottom-left-radius: var(--border-radius-small);
+}
+.control__button:last-of-type:not(.hidden) {
```

The last part of this selector doesn't do anything as far as I can tell.

```suggestion
.control__button:last-of-type {
```

---

that is true. I first tried to use this to exclude the hidden button before changing its position. We can remove it.